pandas/tests/generic/test_generic.py

```diff
@@ -948,3 +947,21 @@ def test_axis_classmethods(self, box):
             assert obj._get_axis_number(v) == box._get_axis_number(v)
             assert obj._get_axis_name(v) == box._get_axis_name(v)
             assert obj._get_block_manager_axis(v) == box._get_block_manager_axis(v)
+
+    # Tests that the ignore keyword causes the intended functionality
```

Comment:
comments go in side the test; add the issue number as a comment

Reply:
Done.